src/library/trackset/setlogfeature.cpp

```diff
@@ -18,6 +18,10 @@
 #include "widget/wlibrarysidebar.h"
 #include "widget/wtracktableview.h"
 
+namespace {
+constexpr int kNumDirectHistoryEntries = 5;
```

Comment:
"direct"? Please use a more descriptive naming for this constant that reveals the purpose.